javaparser-core/src/main/java/com/github/javaparser/metamodel/JavaParserMetaModel.java

```diff
@@ -771,6 +771,8 @@ private static void initializePropertyMetaModels() {
         localRecordDeclarationStmtMetaModel.getDeclaredPropertyMetaModels().add(localRecordDeclarationStmtMetaModel.recordDeclarationPropertyMetaModel);
         returnStmtMetaModel.expressionPropertyMetaModel = new PropertyMetaModel(returnStmtMetaModel, "expression", com.github.javaparser.ast.expr.Expression.class, Optional.of(expressionMetaModel), true, false, false, false);
         returnStmtMetaModel.getDeclaredPropertyMetaModels().add(returnStmtMetaModel.expressionPropertyMetaModel);
+        switchEntryMetaModel.isDefaultPropertyMetaModel = new PropertyMetaModel(switchEntryMetaModel, "isDefault", boolean.class, Optional.empty(), false, false, false, false);
```

Comment:
Did you see this comment "Warning: The content of this class is partially or completely generated - manual edits risk being overwritten"?

Reply:
This change is generated code